step-capstone/src/Components/Hotel.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import {Grid, Typography} from '@material-ui/core'
+
+export default function Hotel(data) {
+
+  /**
+   * Takes data in the form: 
+   * type : 'hotel
+   * name : ' Hotel ZED ' 
+   * location : 'London'
```

Comment:
For mapping purposes, it might be smart to store latitude and longitude as a unique identifier of a location. Hotels don't always have unique names or may not be registered to google maps, so looking up "Hotel ZED in London" might return zero or many results.